### PR TITLE
I Created the Playlist for Session 7 from YouTube

### PR DESCRIPTION
**Pull request by EthanThatOneKid**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,30 @@
 # ColorClassifer-TensorFlow.js
 Community version of TensorFlow.js Color Classifier
+
```
Comment (shiffman) [inline README.md:3]: Could you do this with markdown instead? i.e.

```
* [link title](link url)
```

**Review by shiffman — CHANGES_REQUESTED** [review]
Thank you for contributing! I prefer to have markdown in the README as it's a bit easier to manage. ❤️

File: README.md (hunk 1 above)
Comment (EthanThatOneKid) [reply README.md:3]: I'm on it

**Comment by shiffman** [comment]
This has been taken care of by #5 now! Feel free to submit anything additional if you like!

**Pull request closed without merging**